plugins/wordpress-rule-exclusions-before.conf

```diff
@@ -414,6 +413,20 @@ SecRule REQUEST_FILENAME "@endsWith /wp-admin/user-edit.php" \
             ctl:ruleRemoveTargetById=931130;ARGS:tumblr,\
             ctl:ruleRemoveTargetById=931130;ARGS:youtube,\
             ctl:ruleRemoveTargetById=931130;ARGS:wikipedia,\
+            ctl:ruleRemoveTargetById=931130;ARGS:github,\
+            ctl:ruleRemoveTargetById=931130;ARGS:tiktok,\
+            ctl:ruleRemoveTargetById=931130;ARGS:vkontakte,\
+            ctl:ruleRemoveTargetById=931130;ARGS:medium,\
+            ctl:ruleRemoveTargetById=931130;ARGS:twitter,\
+            ctl:ruleRemoveTargetById=931130;ARGS:tsf-user-meta[facebook_page],\
+            ctl:ruleRemoveTargetById=931130;ARGS:odnoklassniki,\
+            ctl:ruleRemoveTargetById=931130;ARGS:vimeo,\
+            ctl:ruleRemoveTargetById=931130;ARGS:dribbble,\
+            ctl:ruleRemoveTargetById=931130;ARGS:wordpress,\
+            ctl:ruleRemoveTargetByTag=attack-sqli;ARGS:url,\
```

Comment:
Can you move this line next to the `url` argument above?

Reply:
I assume you mean move line 426 below line 407? I can't see the line you highlighted

Reply:
Move it to line 406 or 407 so settings for the same argument (`url` in this case) are placed together.

Reply:
Done